### DISCUSSION TITLE
posts/qxwjgf/

### DISCUSSION THREAD

**Discussion by giscus** [Shitao5/shitao-blog#56] [Comments]
# posts/qxwjgf/

本周末去了小王家，是第一次拜访叔叔阿姨。起因是小王麻麻组织了周六（1109）她团队的团建活动，湖州章里古道徒步，俺和小王也去参加。
[&hellip;] 周五（1108）下班后，收拾收拾衣物，买些水果点心，给建国喂好两天的粮，开上小白便出发。白天看过 55 分钟左右的路程，晚上杭长（宜）高速不知为何封路，去莫干山绕了一圈，多花 20 分钟。按照小王粑粑的指示，把小白停在小王麻麻工作室附近的免费车 &hellip;

https://shitao5.org/posts/qxwjgf/

**Comment by hongtaoh** [comment]
羡慕嫉妒。岳母岳父对诗涛帅哥满意否

**Reply by Shitao5** [reply]
承蒙鸿涛关心，岳父岳母对我蛮和善的，目前一切都顺利嘿嘿！
